etools-searchable-multiselection-menu.html

```diff
@@ -149,6 +149,11 @@
         color: rgba(0,0,0,0.54);
         background-color: #EEEEEE;
       }
+      .emptyValue {
```

Comment:
ok, then, let's keep the label as it is and also not override `.list-wrapper paper-item`